numbat/modules/core/functions.nbt

```diff
@@ -2,64 +2,85 @@ use core::scalar
 
 @name("Identity function")
 @description("Return the input value.")
+@example("id(8kg)")
```

Comment:
Minor nitpick: can we please be consistent with how we format quantities? The official SI version is to have a space between the number and the unit:
```suggestion
@example("id(8 kg)")
```